plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergMetadata.java

```diff
@@ -1194,6 +1194,10 @@ public void executeRemoveOrphanFiles(ConnectorSession session, IcebergTableExecu
                 IcebergConfig.REMOVE_ORPHAN_FILES_MIN_RETENTION,
                 IcebergSessionProperties.REMOVE_ORPHAN_FILES_MIN_RETENTION);
 
+        if (table.currentSnapshot() == null) {
```

Comment:
`remove_orphan_files` procedure still fails on file metastore due to lack of `/data` directory. Could you leave a TODO comment?

Reply:
Do you have a stack trace?

Reply:
The query and stacktrace:
```
trino:tpch> CREATE TABLE test (c1 int);
trino:tpch> ALTER TABLE test EXECUTE remove_orphan_files(retention_threshold => '7d');

Query 20220810_221426_00026_hjz7i failed: Failed accessing data for table: tpch.test
io.trino.spi.TrinoException: Failed accessing data for table: tpch.test
	at io.trino.plugin.iceberg.IcebergMetadata.scanAndDeleteInvalidFiles(IcebergMetadata.java:1255)
	at io.trino.plugin.iceberg.IcebergMetadata.removeOrphanFiles(IcebergMetadata.java:1215)
	at io.trino.plugin.iceberg.IcebergMetadata.executeRemoveOrphanFiles(IcebergMetadata.java:1198)
	at io.trino.plugin.iceberg.IcebergMetadata.executeTableExecute(IcebergMetadata.java:1116)
	at io.trino.plugin.base.classloader.ClassLoaderSafeConnectorMetadata.executeTableExecute(ClassLoaderSafeConnectorMetadata.java:216)
	at io.trino.metadata.MetadataManager.executeTableExecute(MetadataManager.java:345)
	at io.trino.operator.SimpleTableExecuteOperator.getOutput(SimpleTableExecuteOperator.java:128)
	at io.trino.operator.Driver.processInternal(Driver.java:410)
	at io.trino.operator.Driver.lambda$process$10(Driver.java:313)
	at io.trino.operator.Driver.tryWithLock(Driver.java:698)
	at io.trino.operator.Driver.process(Driver.java:305)
	at io.trino.operator.Driver.processForDuration(Driver.java:276)
	at io.trino.execution.SqlTaskExecution$DriverSplitRunner.processFor(SqlTaskExecution.java:740)
	at io.trino.execution.executor.PrioritizedSplitRunner.process(PrioritizedSplitRunner.java:164)
	at io.trino.execution.executor.TaskExecutor$TaskRunner.run(TaskExecutor.java:490)
	at io.trino.$gen.Trino_testversion____20220810_221353_71.run(Unknown Source)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.io.FileNotFoundException: File /var/folders/8s/dkvf18z55lj_9yxhy1n54sph0000gn/T/TrinoTest1711210759979949419/iceberg_data/tpch/test-b3dc0ba83a6542229b672271f21d09eb/data does not exist
	at org.apache.hadoop.fs.RawLocalFileSystem.listStatus(RawLocalFileSystem.java:489)
	at org.apache.hadoop.fs.FileSystem.listStatus(FileSystem.java:1868)
	at org.apache.hadoop.fs.FileSystem.listStatus(FileSystem.java:1910)
	at org.apache.hadoop.fs.FileSystem$4.<init>(FileSystem.java:2072)
	at org.apache.hadoop.fs.FileSystem.listLocatedStatus(FileSystem.java:2071)
	at org.apache.hadoop.fs.ChecksumFileSystem.listLocatedStatus(ChecksumFileSystem.java:700)
	at org.apache.hadoop.fs.FileSystem$5.<init>(FileSystem.java:2183)
	at org.apache.hadoop.fs.FileSystem.listFiles(FileSystem.java:2180)
	at io.trino.plugin.hive.fs.TrinoFileSystemCache$FileSystemWrapper.listFiles(TrinoFileSystemCache.java:386)
	at io.trino.plugin.iceberg.IcebergMetadata.scanAndDeleteInvalidFiles(IcebergMetadata.java:1239)
	... 18 more
```